frontend/src/lib/components/Cards/InsightCard/InsightMeta.tsx

```diff
@@ -122,10 +121,10 @@ export function InsightMeta({
                     {loading && <LemonTableLoader loading={true} />}
                 </>
             }
-            metaDetails={<InsightDetails insight={insight} />}
+            metaDetails={<InsightDetails insight={legacyInsight} />}
```

Comment:
The `<InsightDetails />` component will be refactored in a separate PR as well. It's quite a bit of code and there are bugs present that I'd like to fix while having a look at it.